gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ExtendedFiatService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.services.internal;
+
+import com.netflix.spinnaker.fiat.model.UserPermission;
+import java.util.List;
+import retrofit.http.GET;
+import retrofit.http.Path;
+
+public interface ExtendedFiatService {
```

Comment:
Why is this named `ExtendedFiatService`?

Reply:
Ah just saw `FiatService` which comes from the `fiat-api` module, makes sense.  Did you consider just adding this to `FiatService` in `fiat-api`?

Reply:
we don't currently use it anywhere else, so I figured I would keep it here. fiat-api is mostly supposed to be about making authz decisions (and maybe the thing we want to relocate into kork at some point?)

also I'm not entirely sure that we won't eventually want to push the functionality this is enabling all the way back to an API on fiat (it felt a bit wrong cooking some intimate knowledge of the fiat model into gate..)

Reply:
Alright.  Yeah, we need to pull the fiat-api code out to kork sometime in the next few months I think. 

Yeah, I think pushing the logic of `getServiceAccountsForApplication` to fiat makes sense.  I'm not opposed to starting this way though.